README.md

```diff
@@ -3,15 +3,22 @@ Software necessary to generate WAFS products
 
 To install:
 
-Clone repository (recursively with UPP, if desired):
+Clone repository
 ```bash
-git clone --recursive https://github.com/NOAA-EMC/WAFS
+git clone https://github.com/NOAA-EMC/WAFS
 ```
 
 Move into desired branch and then run:
 
+Clone submodule and sub-submodule repository (including upp and upp/sorc/post_gtg):
+(gtg code is UCAR private, access needs to be authorized)
 ```bash
-./sorc/build_all.sh
+sh sorc/checkout_upp.sh
```

Comment:
Did you forget to add, commit and push this file?

Reply:
Sorry. I thought I would modify it later when adding UPP. Just added it.